jans-auth-server/client/src/test/java/io/jans/as/client/client/Asserter.java

```diff
@@ -51,15 +51,21 @@ public static void assertOk(RegisterResponse response) {
         assertEquals(response.getStatus(), 201, "Unexpected response code: " + response.getEntity());
         assertNotNull(response.getClientId());
         assertNotNull(response.getClientSecret());
-        assertNotNull(response.getClientIdIssuedAt());
         assertNotNull(response.getClientSecretExpiresAt());
+        assertNotNull(response.getRegistrationAccessToken());
     }
 
     public static void assertBadRequest(RegisterResponse registerResponse) {
-        assertEquals(registerResponse.getStatus(), 400, "Unexpected response code");
-        assertNotNull(registerResponse.getEntity());
-        assertNotNull(registerResponse.getErrorType());
-        assertNotNull(registerResponse.getErrorDescription());
+        assertEquals(registerResponse.getStatus(), 400, "Unexpected response code: " + registerResponse.getEntity());
+        assertNotNull(registerResponse.getEntity(), "The entity is null");
+        assertNotNull(registerResponse.getErrorType(), "The error type is null");
+        assertNotNull(registerResponse.getErrorDescription(), "The error description is null");
+    }
+
+    public static void assertBadRequestStatus(RegisterResponse registerResponse) {
```

Comment:
We don't need this method, please remove. It does not differ from `public static void assertBadRequest(RegisterResponse registerResponse)`. If error and error description are not null then indeed entity can't be null as well because otherwise from where it took error and description.

Reply:
method was removed.